sources/packages/web/src/components/common/StudentApplication.vue

```diff
@@ -256,14 +257,24 @@ export default defineComponent({
           SELECTED_OFFERING_END_DATE_KEY,
         );
       }
-      // If the user after selecting a study period finds that
-      // they need to check my study period not listed, then
-      // the details of previously selected
-      // study period must be cleared.
+      if (
+        event.changed?.component.key === LOCATIONS_DROPDOWN_KEY &&
+        +event.changed.value > 0
```

Comment:
There is a + sign here. Is it a typo?

Reply:
Just checked. `event.changed.value` is already a number. So, removed `+`. Thanks.

Reply:
Thanks Shashank. I now know that + is to convert string to number. We can keep + as it is.